screenshot-test/App.screenshot.js

```diff
@@ -0,0 +1,596 @@
+const VIEW_PORT_1080 = {
+  width: 1920,
+  height: 1080,
+  deviceScaleFactor: 1,
+}
+const WAITING_TIME = 2000;
+const LONG_WAITING_TIME = 10000;
+const LATEST_SNAPSHOTS_DIR = './screenshot-test/__latest_snapshots__/';
+const browserConfig = { // Docker requires --no-sandbox to be able to run the tests
+  headless: true,
+  args: ['--no-sandbox', '--disable-setuid-sandbox'],
+  ignoreHTTPSErrors: true,
+  dumpio: false
+}
+
+const apiAccount = fs.readFileSync(`${DATA_DIR}api-account.json`).toString();
+const apiV1Info = fs.readFileSync(`${DATA_DIR}api-v1-info.json`).toString();
+const numbersLevels = fs.readFileSync(`${DATA_DIR}private-utils-numbers-levels.json`).toString();
+const numbersMain = fs.readFileSync(`${DATA_DIR}private-utils-numbers-main.json`).toString();
+const geneList = fs.readFileSync(`${DATA_DIR}utils-cancerGeneList.json`).toString();
+const allCuratedGenes = fs.readFileSync(`${DATA_DIR}utils-allCuratedGenes.json`).toString();
+const evidenceLevels = fs.readFileSync(`${DATA_DIR}private-utils-evidences-levels.json`).toString();
+const tumorTypes = fs.readFileSync(`${DATA_DIR}private-utils-tumorTypes.json`).toString();
+const geneNumbers = fs.readFileSync(`${DATA_DIR}private-utils-numbers-gene-ABL1.json`).toString();
+const geneQuery = fs.readFileSync(`${DATA_DIR}api-v1-genes-ABL1.json`).toString();
+const biologicalVariants = fs.readFileSync(`${DATA_DIR}api-private-search-variants-bio-ABL1.json`).toString();
+const clinicalVariants = fs.readFileSync(`${DATA_DIR}api-private-search-variants-cli-ABL1.json`).toString();
+const sampleCount = fs.readFileSync(`${DATA_DIR}api-private-utils-portalAlterationSampleCount.json`).toString();
+const evidenceSummary = fs.readFileSync(`${DATA_DIR}api-v1-evidences-ABL1-summary.json`).toString();
+const evidenceBackground = fs.readFileSync(`${DATA_DIR}api-v1-evidences-ABL1-background.json`).toString();
+const variantAnnotation = fs.readFileSync(`${DATA_DIR}api-private-utils-variantAnnotation-ABL1-BCR.json`).toString();
+const genomenexusTranscript = fs.readFileSync(`${DATA_DIR}genomenexus-transcript.json`).toString();
+const genomenexusCanonicalTranscript = fs.readFileSync(`${DATA_DIR}genomenexus-canonical-transcript.json`).toString();
+const userSize = fs.readFileSync(`${DATA_DIR}api-users-size.json`).toString();
+const userDetails = fs.readFileSync(`${DATA_DIR}api-users-details.json`).toString();
+
+// # Fix the time to expiration date. 
+function updateTokenExpirationDate(current){
+  let today = new Date()
+  today.setDate(today.getDate()+100);
+  current.forEach(token => {
+    token.expiration = today.toISOString();
+  })
+  return current;
+}
+let apiAccountToken = fs.readFileSync(`${DATA_DIR}api-account-token.json`).toString();
+apiAccountToken = JSON.stringify(updateTokenExpirationDate(JSON.parse(apiAccountToken)));
+let userToken = fs.readFileSync(`${DATA_DIR}api-users-tokens.json`).toString();
+userToken = JSON.stringify(updateTokenExpirationDate(JSON.parse(userToken)));
```

Comment:
Add a process to set expiration date dynamically, or the screenshots of Account Settings Page and User Details Page will be different daily(because of to expiration date).